client/src/components/CategoryBarImage/CategoryBarImage.module.scss

```
@@ -0,0 +1,28 @@
+@import "../../global.scss";
+
+.CategoryBarImage {
+    display: flex;
+    justify-content: space-around;
+    margin: 1rem 0;
+
+    a {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        font-family: $ff-roboto;
+        font-size: $fz-small-4;
+        color: $fc-3;
+
+        img {
+            width: 10rem;
+            border-radius: 50%;
```

Border radius 50% funktioniert solang du ein komplett quadratisches Bild hast. sobald dem nicht mehr so ist, bekommst du ein EI. 
Wenn du das vermeiden willst kannst du einfach einen exorbitant hohen Wert nehmen.
```suggestion
            border-radius: 999vw;
```